### ISSUE TITLE
Support libvirt vagrant provider

### ISSUE THREAD

**Issue by olberger** [nre-learning/antidote-selfmedicate#47]
It could be interesting to be able to use libvirt (for qemu/kvm) on Linux as an alternative to virtualbox.

Would be interesting to verify any gain on performance...

**Comment by olberger** [comment]
It works !

I have committed in my libvirt branch the needed change: https://github.com/olberger/antidote-selfmedicate/commit/69b5d173f6b2200dff0ddd4154962b9c7860b870
Basically all there is to it is using a Vagrant base box built for libvirt :-)

**Comment by smk4664** [comment]
@olberger Vagrant removed libvirt, are you using a plugin to add libvirt support back in?  Would you mind linking documentation on how you were able to get this running?

**Comment by olberger** [comment]
I guess I must be using https://github.com/vagrant-libvirt/vagrant-libvirt 

I think I saw mentions of @cloudtoad struggling with it on twitter yesterday, and maybe he could confirm about the necessary docs additions ?
Of course I can try and add some useful bits.
Maybe something along those lines: https://github.com/olberger/vagrant-k8s-kubevirt-katacontainers#install-pre-requisites (not mine originally) ... 

and by the way, you might be interested  to have a look at this repo, which demos how to secure containers with KataContainers (re MP3)

**Comment by olberger** [comment]
Of course, another option could be to just use minikube which is able to use libvirt directly, when on Linux and if qemu/kvm is preferred... but that reverses the point of using Vagrant genrally :-/

Or we could try Vagrant with no virtualizer to start minikube with qemu... if that's even feasable ?...

Too many options ;)

**Comment by olberger** [comment]
I'm a bit doubtful on the last one, i.e. vagrant doing pretty much nothing... sorry.

**Comment by blinklet** [comment]
I added the libvirt provider to the vagrantfile and it works well. You need to update antidote-config.yml to set the provider to libvirt and you need to install the vagrant-libvirt plugin. 

See: https://github.com/nre-learning/antidote-selfmedicate/pull/62
